source/slang/core.meta.slang

```diff
@@ -2823,6 +2823,9 @@ attribute_syntax [fastopt]              : FastOptAttribute;
 __attributeTarget(LoopStmt)
 attribute_syntax [allow_uav_condition]  : AllowUAVConditionAttribute;
 
+__attributeTarget(LoopStmt)
+attribute_syntax [maxiters(count)]      : MaxItersAttribute;
```

Comment:
The naming convention for new attributes is BigCamel. The existing all lower case attribute names are for HLSL compatibility.